src/main/kotlin/org/cascadebot/cascadebot/utils/Extensions.kt

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2020 CascadeBot. All rights reserved.
+ * Licensed under the MIT license.
+ */
+
+package org.cascadebot.cascadebot.utils
+
+import kotlin.math.pow
+import kotlin.math.round
+import kotlin.math.roundToInt
+
+
+fun String.truncate(length: Int, ellipses: Boolean = true): String =
```

Comment:
We need to find a way to incorporate this into language as `...` is not a thing is some languages

Reply:
eh...

Reply:
This is such a specific case that we don't really need to handle this right now (As discussed)